fud/fud/exec.py

```diff
@@ -101,7 +101,11 @@ def run_fud(args, config):
         for ed in path:
             sp.start_stage(f"{ed.stage.name} → {ed.stage.target_stage}")
             try:
-                result = ed.stage.run(data, sp)
+                if getattr(ed.stage, "_no_spinner", False):
```

Comment:
NBD, but if you want, you could put a default `_no_spinner` value in the base `Stage` class to avoid the need for a `getattr` (and just use `ed.stage._no_spinner` instead).